olga.yepikhina/lecture-5/contactus.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<meta charset="UTF-8">
+<html>
+
+<head>
+  <link rel="stylesheet" href="contactus.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Roboto+Slab" rel="stylesheet">
```

Comment:
Those I assume (not 100%) may be possible to import in the CSS. Not sure 100% tho :)